pyproject.toml

```diff
@@ -26,6 +26,7 @@ dependencies = [
   "rich>=13.7,<13.10",
   "sqlglot>=26.6.0,<27.0.0",
   "duckdb==1.1.2",
+  "fsspec",
```

Comment:
Where is this used?

Reply:
In file `datacontract\engines\soda\connections\duckdb.py`, the method `sniff_csv_header` uses `duckdb.from_csv_auto` to read a csv file as a stream.

Without `fsspec`, the following fails:

```
return duckdb.from_csv_auto(io.BytesIO(header_line), **csv_params).columns
E       duckdb.duckdb.InvalidInputException: Invalid Input Error: This operation could not be completed because required module 'fsspec' is not installed
```